packages/backend-modules/documents/graphql/schema-types.js

```diff
@@ -107,9 +107,8 @@ type Meta {
   shareBackgroundImageInverted: String
 
   credits: JSON
-
-  authors: [User!]! @deprecated(reason: "use \`Meta.contributors\` instead")
```

Comment:
is this something to worry about for the frontend? did you check whether `authors` is in use somewhere?